src/uu/ls/src/ls.rs

```diff
@@ -1921,10 +1925,17 @@ pub fn list(locs: Vec<&Path>, config: &Config) -> UResult<()> {
                 }
                 writeln!(out, "{}:", path_data.p_buf.display())?;
                 if config.dired {
-                    dired::calculate_subdired(&mut dired, path_data.display_name.len());
+                    // Fist directory displayed
+                    let dir_len = path_data.display_name.len();
+                    // add the //SUBDIRED// coordinates
+                    dired::calculate_subdired(&mut dired, dir_len);
+                    // Add the padding for the dir name
+                    dired::add_dir_name(dir_len, &mut dired);
```

Comment:
Looks a bit odd to use different function signatures for these two functions.

Reply:
good point, I will fix it